contrib/openshift/skydive-template.yaml

```diff
@@ -167,9 +156,12 @@ objects:
           env:
           - name: SKYDIVE_ANALYZERS
             value: $(SKYDIVE_ANALYZER_SERVICE_HOST):$(SKYDIVE_ANALYZER_SERVICE_PORT_API)
-          envFrom:
-          - configMapRef:
-              name: skydive-agent-config
+          - name: SKYDIVE_LOGGING_LEVEL
+            value: ${SKYDIVE_LOGGING_LEVEL}
+          - name: SKYDIVE_AGENT_TOPOLOGY_PROBES
+            value: "ovsdb runc docker"
+          - name: SKYDIVE_RUNC_PATH
+            value: "/run/containerd/runc /var/run/runc /var/run/runc-ctrs"
```

Comment:
The `/run/containerd/runc` is not mapped

Reply:
Indeed, thangs. Changed.